html/about.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About TEDx</title>
+    <!-- link to external resources -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <!-- link to other files -->
+    <link rel="stylesheet" href="/css/style.css" />
+    <link rel="stylesheet" href="/css/about.css" />
+    <script src="/js/about.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <div class="d-navbar flex g-padding">
+        <a href="/index.html">
+          <img class="logo" src="/images/tedx-logo.jpg" alt="TEDx's Logo" />
```

Comment:
- [ ] All paths (src, href) in the about.html file are broken. I guess you carried this file into the HTML folder after creating it. Currently, I can only see basic HTML content for this file. 

Screenshot
![Screenshot from 2021-09-16 21-23-09](https://user-images.githubusercontent.com/1642014/133665758-228f750d-fd2a-44c9-88f7-373b8bb015b6.png)
o

Reply:
I don't know why. they are working fine on my pc. I have double-checked it. I have not moved about.html anywhere neither any other file.

Reply:
Hi @shadabwahidullah, 
Different operating systems and web servers may resolve paths differently. It may work on your system but you should consider all devices and use standard syntax. 
```
.(a single dot) - this represents the current directory.
..(two dots) - this represents the parent directory. 
```


For example, this (`/images/tedx-logo.jpg`) path may work on vscode live server windows but it will not work on Unix (which most of the internet servers) devices. You should give the correct path to the external file instead of relying on the web server to find it.
 

`/images/tedx-logo.jpg` this should be `../images/tedx-logo.jpg`

Reply:
Thank you for your answer. I will keep that in mind.